languages/javalang/builder-renderer/java-builder-client/src/main/kotlin/io/vrap/codegen/languages/javalang/client/builder/requests/JavaHttpRequestRenderer.kt

```diff
@@ -155,25 +155,25 @@ class JavaHttpRequestRenderer @Inject constructor(override val vrapTypeProvider:
                 .map { "this.$it" }
                 .joinToString(separator = ", ")
 
-        val bodyName: String? = if (this.bodies != null && this.bodies.isNotEmpty()) {
-            if (this.bodies[0].type.toVrapType() is VrapObjectType) {
+        val bodyName : String? = if(this.bodies != null && this.bodies.isNotEmpty()){
+            if(this.bodies[0].type.toVrapType() is VrapObjectType) {
                 val methodBodyVrapType = this.bodies[0].type.toVrapType() as VrapObjectType
                 methodBodyVrapType.simpleClassName.decapitalize()
-            } else {
+            }else {
                 "jsonNode"
             }
-        } else {
+        }else {
             null
         }
-
-        val addingQueryParams: String = this.queryParameters
-                .filter { it.annotations.find { it.type.name.equals("placeholderParam") } == null }
-                .map { "params.add(this.${it.fieldName()}.stream().map(s -> \"${it.name}=\" + ${if (it.type.name.equals("string")) "urlEncode(s)" else "s"}).collect(Collectors.joining(\"&\")));" }
+        
+        val addingQueryParams : String = this.queryParameters
+                .filter { it.annotations.find { it.type.name.equals("placeholderParam") } == null}
+                .map { "params.add(this.${it.fieldName()}.stream().map(s -> \"${it.name}=\" + ${if(it.type.name.equals("string")) "urlEncode(s)" else "s"}).collect(Collectors.joining(\"&\")));" }
```

Comment:
Can we somehow make this easier to read? It's really difficult for me to understand this code...